### ISSUE TITLE
Not an issue, really.

### ISSUE THREAD

**Issue by Sanskar7964** [abdullahhussein1/ChatApp#3]
Hi, I wanted to ask if it is possible to convert this app to a support chat feature? I want to add a admin panel with role based access control, which allows support team to chat and answer site users. Any guidance would be of great help.

**Comment by abdullahhussein1** [comment]
Hi @Sanskar7964, I think it would be easy, you can set the chat destination to your admin panel, any sended message by user will show inside the admin panel and there you can answer them seperately (here you can use the current interface).

is it clear?

**Comment by Sanskar7964** [comment]
Yes, thank you. I need to create a separate UI for admin/support team, too?

**Comment by abdullahhussein1** [comment]
> Yes, thank you. I need to create a separate UI for admin/support team, too?

No I think it's not necessary, but why do you think it should be role based?

**Comment by Sanskar7964** [comment]
I think, because I will need admin to manage the staff and customers, and staff to have their own window to chat with customers. I think both can't be using same UI to communicate

**Comment by abdullahhussein1** [comment]
You are right
